keps/NNNN-kep-template/README.md

```diff
@@ -270,6 +267,61 @@ when drafting this test plan.
 [testing-guidelines]: https://git.k8s.io/community/contributors/devel/sig-testing/testing.md
 -->
 
+[ ] I/we understand the owners of the involved components may require updates to
+existing tests to make this code solid enough prior to committing the changes necessary
+to implement this enhancement.
+
+##### Prerequisite testing updates
+
+<!--
+Based on reviewers feedback describe what additional tests need to be added prior
+implementing this enhancement to ensure the enhancements have also solid foundations.
+-->
+
+##### Unit tests
+
+<!--
+In principle every added code should be unit tested. However, the exact tests are hard
+to answer ahead of implementation. As a result, this section should only be filled when
+targeting Beta and GA to ensure that something wasn't missed during Alpha implementation.
+
+Talking about individual tests would be an overkill, so we just require to list the
+packages that were touched during the implementation together with their current test
+coverage in the form:
+- <package>: <current test coverage>
```

Comment:
I'm definitely happy to add what you suggested above.

But I don't really agree that the coverage information is a wrong time/place for this information.
If you're worried about staleness - let me change that to:
- <package>: <date> - <coverage>

This will be definition not go stale. And having the information about the coverage will expose it to people just reading KEPs and not watching to individual PRs to see how it goes.

My main reason for ensuring it's put here is that KEP approvers are in many cases not approvers for the code itself (or at least not all the code). So putting it here verbatim will show it very clearly into their eyes.

Reply:
to be clear about the timing aspect... I think pre-alpha/alpha is exactly the right time to be asking this question, since that's when code starts merging and maybe breaking existing undertested stuff

Reply:
> to be clear about the timing aspect... I think pre-alpha/alpha is exactly the right time to be asking this question, since that's when code starts merging and maybe breaking existing undertested stuff

OK - so I'm all for asking that for Alpha (if you remember that was actually my initial proposal).
But your couterargument is that the exact set of packages will sometimes be hard to predict.

So before Alpha is the compromise because:
(a) a significant part of implementation is already done at this point
(b) it still isn't enabled by default, which means that enhancement owner still have big motivation to move it to Beta